### DISCUSSION TITLE
DIA NN alone and MSStats

### DISCUSSION THREAD

**Discussion by RamonDiazPenya1** [vdemichev/DiaNN#990] [General]
Hi there!

Is it possible to use DIA NN alone (not the fragger pipeline) in combination with MSStats?

Thanks!

**Comment by vdemichev** [comment]
You can definitely run DIA-NN alone. The output should be in the same format as when running from within the FragPipe, hence compatible with the same downstream processing.

**Comment by RamonDiazPenya1** [comment]
Thanks for the answer!

**Comment by RamonDiazPenya1** [comment]
Hi Again,
I checked the output files from DIANN alone, and there is no file Mstats.cvs (or similar) which is generated when using DIANN-Fragger. Is this possible or I´m wrong?  Please find herein attached some lines of Mstats.cvs as exemple.
[msstats_fewlines.csv](https://github.com/vdemichev/DiaNN/files/15002596/msstats_fewlines.csv)

**Reply by vdemichev** [reply]
Yes, this kind of format is not produced by DIA-NN. Not sure how FragPipe generates this

**Comment by RamonDiazPenya1** [comment]
Thanks again Vadim!
